src/client/components/pages/parts/editor-achievements.js

```diff
@@ -125,19 +125,11 @@ class EditorAchievementTab extends React.Component {
 						<DragAndDrop name="rank3"/>
 					</div>
 					<div className="form-group">
-						<span>
-							<button
-								className="btn btn-default"
-								type="submit"
-							>
+						<span style={{marginLeft: '30px'}}>
+							<button className="btn btn-default" type="submit">
```

Comment:
I think this would benefit from being a `btn-primary`.

Reply:
well... `btn-default` goes very well the color-theme of bookbrainz (brown, orange).
But if we want to highlight it here, we can go with `btn-success`, same as the search button.
Let me know.

Reply:
Yes `btn-success` is the right choice.